docs/htdocs/info/docs/tools/vep/script/vep_example.html

```diff
@@ -236,6 +236,43 @@ <h2 id="dbNSFP">dbNSFP</h2>
   sequence used to calculate the scores in dbNSFP. </p>
   
   <hr/>
+
+
+  <h2 id="StructVar">Structural Variants</h2>
+
+  <p>VEP can be used to annotate structural variants with their predicted effect on other genomic features.
+     Input data should be supplied as VCF.
+  </p>
+
+  <p> <b>Prediction process</b>
+  <ul>
+  <li> The INFO keys 'END' or 'SVLEN' are present, the proportion of any overlapping feature covered by the variant is calculated
+  <li> If the SVTYPE or ALT is 'DEL', the variant tested for feature ablation/ truncation
+  <li> If the SVTYPE or ALT is 'DUP', the variant tested for feature amplification
+  <li> If the SVTYPE or ALT is 'INS' or 'DUP', the variant tested for feature elongatation
+  <li> SVTYPE is used in preference to ALT to derive the variant type of an SV with 'CN*' alleles
+  </ul></p>
+
+  <p><b> Reported overlaps</b>
+  <ul>
+  <li> VEP calculates the length and proportion of each genomic feature overlapped by a structural variant
+  <li> Use the <a href="vep_options.html#opt_overlaps">overlaps</a> option to enable this when using VCF or tab format.
```

Comment:
We used to display the options with the `--` in front of them, e.g. `--overlaps`